data_structures/bloom_filters/bloom_filter/main.c

```diff
@@ -0,0 +1,21 @@
+#include "./bloom_filter.h"
+#include <sys/resource.h>
```

Comment:
I am pretty certain sys/resource.h is a POSIX header and not a part of the C standard library, I didn't spot anything that actually depends on it anyway
```suggestion
#include "bloom_filter.h"
```